.gitignore

```diff
@@ -13,3 +13,4 @@ centos_package_cron_src.tgz
 centos-package-cron.spec
 .bundle
 Gemfile.lock
+deploy.sh
```

Comment:
@mrwilson or @nryn - Is this an accidental commit?

Reply:
This is accidental, we'll roll this back.